tensorflow/lite/micro/examples/micro_speech/simple_features/CEVA_BlueVox/fft.cc

```diff
@@ -0,0 +1,99 @@
+
+
+
```

Comment:
I don't believe this file is used, it's legacy, instead the one in micro_features implements the fft now used in the example.